### ISSUE TITLE
~100MB log is installed using yarn/npm (probably using others also)

### ISSUE THREAD

**Issue by bambooCZ** [newrelic/node-newrelic#368]
## Description

`node_modules` are already considered to be the [heaviest object in the universe](https://www.reddit.com/r/ProgrammerHumor/comments/6s0wov/heaviest_objects_in_the_universe/?utm_source=share&utm_medium=web2x). You guys add another dimension to it :). On clean installation there is ~100MB log included.

## Steps to Reproduce

```
mkdir my-new-project
cd my-new-project
npm init
npm i newrelic --only=production
ls -lh node_modules/newrelic
```

## Expected Behavior

Logs shouldn't be installed by any package manager. Especially logs that are this big.

## Relevant Logs / Console output
```
$ ls -lh node_modules/newrelic/
total 107M
-rw-r--r--  1 usr grp  51K 26. říj  1985  api.js
drwxr-xr-x  2 usr grp 4,0K  2. čen 03.35  bin
-rw-r--r--  1 usr grp 3,9K 26. říj  1985  CONTRIBUTING.md
-rw-r--r--  1 usr grp 5,6K 26. říj  1985  index.js
-rw-r--r--  1 usr grp  337 26. říj  1985  jsdoc-conf.json
drwxr-xr-x 19 usr grp 4,0K  2. čen 03.35  lib
-rw-r--r--  1 usr grp  20K 26. říj  1985  LICENSE
-rw-r--r--  1 usr grp  12K 26. říj  1985 'Migration Guide.md'
-rw-r--r--  1 usr grp 106M 26. říj  1985  newrelic_agent.log
-rw-r--r--  1 usr grp 1,5K 26. říj  1985  newrelic.js
-rw-r--r--  1 usr grp 182K 26. říj  1985  NEWS.md
-rw-r--r--  1 usr grp 6,3K  2. čen 03.35  package.json
-rw-r--r--  1 usr grp  31K 26. říj  1985  README.md
-rw-r--r--  1 usr grp  719 26. říj  1985  SECURITY.md
-rw-r--r--  1 usr grp 2,9K 26. říj  1985  stub_api.js
-rw-r--r--  1 usr grp  34K 26. říj  1985  THIRD_PARTY_NOTICES.md
```
Do you see the `newrelic_agent.log` :)

**Comment by michaelgoin** [comment]
Oof. Guessing that is only in the latest version and would be my bad.

Can you confirm this is `v6.8.0`?

Deployed from a new location and apparently missed the log in there when I scanned the dry run. Another case for using the "files" list eventually.

When we get a new release out, I'll probably deprecate this one.

**Comment by bambooCZ** [comment]
Yup it is 6.8.0.

I have tried 6.7.0 and 6.7.1 and both 6.7.x seems OK.

**Comment by michaelgoin** [comment]
@bambooCZ thank you for confirming and letting us know.

Sorry about that!

**Comment by bambooCZ** [comment]
I am glad to help :)

**Comment by astormnewrelic** [comment]
@bambooCZ -- we published the latest agent (6.9.0) _without_ this accidental artifact. We're closing this out for now, but if you're still seeing a problem please don't hesitate to let us know.  Thanks again for letting us know about this and helping to make the agent a little bit better.
